### DISCUSSION TITLE
LXQt 1.*.*  Release Articles

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2322] [Media]
* https://www.phoronix.com/news/LXQt-1.2-Released (most comments about wayland)
* https://9to5linux.com/lxqt-1-2-desktop-environment-released-with-initial-wayland-support-various-improvements (they were faster than we tweeting about it)
* https://linuxiac.com/lxqt-1-2-desktop-environment-released-this-is-whats-new/

Some other in spain, turkish and czech

**Comment by tsujan** [comment]
> most comments about wayland

Many users are confused about Wayland and think it's just about another component to be added. I wouldn't take their comments seriously.

**Comment by nlsmart** [comment]
Hi,

Congratulation to the dev.

**Comment by stefonarch** [comment]
Another one, with many screens: https://www.debugpoint.com/lxqt-1-2-0-features/

**Reply by tsujan** [reply]
> Another one, with many screens: https://www.debugpoint.com/lxqt-1-2-0-features/

The reviewer has paid attention to details. Just one small mistake: 1.2.0 isn't a point release.

**Comment by darkshram** [comment]
[https://www.alcancelibre.org/noticias/disponible-lxqt-1-2-0](https://www.alcancelibre.org/noticias/disponible-lxqt-1-2-0)

This post (by me) is a non-verbatim translation to spanish of the release notes for 1.2.0 release.

**Comment by darkshram** [comment]
https://www.alcancelibre.org/noticias/disponible-lxqt-1-3-0

This [other] post (by me) is a non-verbatim translation to Spanish of the release notes for LXQt 1.3.0.

**Comment by stefonarch** [comment]
> popular lightweight desktop environment LXQt 

https://www.debugpoint.com/lxqt-1-3-0/

**Comment by edwardp3** [comment]
Any thoughts on when Debian Sid (unstable) might receive the 1.3.0 packages? Debian's pkg-lxqt-devel mailing list, contains mostly spam, since the end of January,

Thanks.

**Reply by stefonarch** [reply]
afaik bookworm is in freeze now, so in summer when it will be the new stable probably?

**Reply by edwardp3** [reply]
Hope so.

**Reply by edwardp3** [reply]
I opened a bug report to have the 1.3.0 packages added to Fedora Rawhide. I was told by one of their forum admins that a lot of the packages in Fedora now have it set up to automatically file a bug report when a new version is released, but the admin did not see that for LXQt.
